components/ui-shell/index.scss

```diff
@@ -0,0 +1,241 @@
+@import "@carbon/type/scss/type";
+@import "@carbon/colors/scss/colors";
+
+// These are needed to extend default carbon component styles
+@import "carbon-web-components/scss/components/ui-shell/header";
+@import "carbon-web-components/scss/components/ui-shell/side-nav";
+
+// These are needed to apply theming
+@import "carbon-components/scss/globals/scss/css--helpers";
+@import "carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/themes/mixins";
+
+$q: "qiskit";
+$prefix: "bx";
+
+:host {
+  --header-height: 3.5rem;
+
+  @include carbon--theme($carbon--theme--white, true);
+
+  font-family: carbon--font-family("sans");
+}
+
+.#{$q}-header {
+  display: flex;
+  justify-content: space-between;
+  height: var(--header-height);
+  border-color: var(--cds-border-subtle);
+  background-color: var(--cds-ui-background);
```

Comment:
thank you for including a bg-color here, I _think_ this could solve the need from this bug #137